sound/soc/sof/loader.c

```diff
@@ -197,6 +197,29 @@ static int ext_man_get_dbg_abi_info(struct snd_sof_dev *sdev,
 	return 0;
 }
 
+static int ext_man_get_config_data(struct snd_sof_dev *sdev,
+				   const struct sof_ext_man_elem_header *hdr)
+{
+	const struct sof_ext_man_config_data *config_data =
+		container_of(hdr, struct sof_ext_man_config_data, hdr);
+	u32 ipc_size;
+	int i;
+
+	for (i = 0; i < SOF_EXT_MAN_CONFIG_LAST_ELEM - 1; i++)
+		switch (config_data->elems[i].token) {
+		case SOF_EXT_MAN_CONFIG_IPC_MSG_SIZE:
+			ipc_size = config_data->elems[i].value;
```

Comment:
and it isn't used? What's the purpose of this manifest entry? Shouldn't it affect IPC processing?

Reply:
@fredoh9 I'll also echo. It doesn't really make sense to add functions that are not used. Have you looked at how to hook this up to IPC code to adjust the max size there?